playwright.config.ts

```diff
@@ -5,7 +5,7 @@ dotenv.config();
 
 const config: PlaywrightTestConfig = {
   testDir: './e2e/tests',
-  timeout: 10 * 60 * 1000,
```

Comment:
Note: The Keycloak-Odoo streaming requires a longer timeout due to its configuration, so I reduced the global timeout to 3 minutes and set longer timeouts for the relevant test cases.